app/src/main/java/com/doctor/yumyum/presentation/ui/myrecipe/MyRecipeFragment.kt

```diff
@@ -72,6 +78,29 @@ class MyRecipeFragment : BaseFragment<FragmentMyRecipeBinding>(R.layout.fragment
         sortSelectDialog.setContentView(sortSelectBinding.root)
     }
 
+    private fun initRecycler() {
+        val myRecipeListAdapter = ResearchListAdapter({ recipeId ->
+            val intent = Intent(requireContext(),RecipeDetailActivity::class.java)
+            intent.putExtra("recipeId", recipeId)
+            startActivity(intent)
+        },{})
+        binding.myRecipeRvPost.adapter = myRecipeListAdapter
+
+        myRecipeViewModel.myRecipeList.observe(viewLifecycleOwner){
+            myRecipeListAdapter.setRecipeList(it)
+        }
+    }
+
+    private fun startForSort() {
+        sortLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+            //TODO : 정렬 정보 가져오기
+        }
+        binding.myRecipeIbSort.setOnClickListener {
+            val intent = Intent(context, MyPageFilterActivity::class.java)
+            sortLauncher.launch(intent)
+        }
+    }
```

Comment:
정렬은 다이얼로그로 처리하고 여기서는 필터 처리를 하는 것 같은데 이름을 바꿔야 할 것 같습니다
```suggestion
    private fun startForFilter() {
        filterLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
            //TODO : 필터 정보 가져오기
        }
        binding.myRecipeIbFilter.setOnClickListener {
            val intent = Intent(context, MyPageFilterActivity::class.java)
            filterLauncher.launch(intent)
        }
    }
```